static/js/reactions.js

```diff
@@ -236,22 +236,37 @@ exports.view.update_existing_reaction = function (opts) {
     // simply updates the DOM.
 
     var message_id = opts.message_id;
-    var emoji_name = opts.emoji_name;
     var user_list = opts.user_list;
     var user_id = opts.user_id;
     var local_id = exports.get_local_reaction_id(opts);
     var reaction = exports.find_reaction(message_id, local_id);
 
     exports.set_reaction_count(reaction, user_list.length);
 
-    var new_title = generate_title(emoji_name, user_list);
-    reaction.prop('title', new_title);
-
     if (user_id === page_params.user_id) {
         reaction.addClass("reacted");
     }
 };
 
+exports.generate_popover = function (elem, message_id, local_id) {
+    // Add a popover showing who reacted to a message.
+    var message = get_message(message_id);
+    var user_list = get_user_list_for_message_reaction(message, local_id);
+    var emoji_name = exports.get_reaction_info(local_id).emoji_name;
+    var title = generate_title(emoji_name, user_list);
+    var html_title = '<div>'+title+'</div>';
+    var ypos = elem.offset().top;
+    elem.popover({
```

Comment:
I think you would like to check if the popover already exists and create one only if none exist(popover removal calls are asynchronous), right?